### ISSUE TITLE
No GUI

### ISSUE THREAD

**Issue by FelixBl** [manly-man/Crowser#1]
i dont want Crowser to have a graphical user interface!

**Comment by deubzerm** [comment]
yeah i think a CLI with an option to use it with a joystick would be sufficient

**Comment by halbtuerke** [comment]
There will never be a GUI!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by FelixBl** [comment]
Any updates?
